### DISCUSSION TITLE
微信开发工具使用LIN-UI image-clipper组件时在PC端提示错误信息，在手机端可以正常使用

### DISCUSSION THREAD

**Discussion by jerryliushilin** [TaleLin/lin-ui#1258] [求助] [answered]
错误信息如下:
[渲染层网络层错误] Failed to load local image resource /miniprogram_npm/lin-ui/image-clipper/images/photo.png 
 the server responded with a status of 500 (HTTP/1.1 500 Internal Server Error) 

[渲染层网络层错误] Failed to load local image resource /miniprogram_npm/lin-ui/image-clipper/images/rotate-along.png 
 the server responded with a status of 500 (HTTP/1.1 500 Internal Server Error) 

[渲染层网络层错误] Failed to load local image resource /miniprogram_npm/lin-ui/image-clipper/images/rotate-inverse.png 
 the server responded with a status of 500 (HTTP/1.1 500 Internal Server Error) 

[渲染层网络层错误] Failed to load local image resource /miniprogram_npm/lin-ui/image-clipper/images/sure.png 
 the server responded with a status of 500 (HTTP/1.1 500 Internal Server Error) 

[渲染层网络层错误] Failed to load local image resource /miniprogram_npm/lin-ui/image-clipper/images/close.png 
 the server responded with a status of 500 (HTTP/1.1 500 Internal Server Error) 
在PC上使用此组件报错，在手机端使用是正常的

**Comment by murongg** [comment]
请问使用的 lin-ui 是什么版本

**Reply by jerryliushilin** [reply]
lin-ui使用的是0.8.6版本，但是image-clipper组件更新到0.8.13版本了，更新后一直使用正常，就是最近使用报错

**Reply by murongg** [reply]
我使用最新版本的微信开发者工具与最新的基础库，没有复现此问题

**Reply by jerryliushilin** [reply]
我也是使用的最新微信开发者工具，和最新的基础库2.14.4，您看我是否把lin-ui的版本升级一下？

**Reply by murongg** [reply]
建议新开一个项目安装最新版本的lin-ui，测试没有问题后再把自己的项目升级

**Reply by jerryliushilin** [reply]
我新开了一个项目，版本都是最新的，还是报错，错误如下：
页面【miniprogram_npm/lin-ui/calendar/index]错误:
 Error: module "miniprogram_npm/lin-ui/common/async-validator/validator/type.js" is not defined
    at require (VM282 WAService.js:2)
    at VM282 WAService.js:2
    at index.js? [sm]:1
    at require (VM282 WAService.js:2)
    at VM282 WAService.js:2
    at index.js? [sm]:1
    at require (VM282 WAService.js:2)
    at VM282 WAService.js:2
    at rules.js? [sm]:1
    at require (VM282 WAService.js:2)
(anonymous) @ VM408:348
doWhenAllScriptLoaded @ VM407:1040
scriptLoaded @ VM407:1060
(anonymous) @ VM407:1085
Promise.then (async)
(anonymous) @ VM407:1111
(anonymous) @ VM407:1115
loadScripts @ assubloader.js:1
async function (async)
loadScripts @ assubloader.js:1
loadAppResourceScripts @ assubloader.js:1
(anonymous) @ VM402:1
t.loadScripts @ VM16 asdebug.js:1
loadAppService @ VM16 asdebug.js:1
messagerCallback @ VM16 asdebug.js:1
(anonymous) @ VM16 asdebug.js:1
f @ VM16 asdebug.js:1
g @ VM16 asdebug.js:1
(anonymous) @ VM16 asdebug.js:1
_ws.onmessage @ VM16 asdebug.js:1
VM282 WAService.js:2 Error: module "miniprogram_npm/lin-ui/common/async-validator/validator/type.js" is not defined
    at require (VM282 WAService.js:2)
    at VM282 WAService.js:2
    at index.js? [sm]:1
    at require (VM282 WAService.js:2)
    at VM282 WAService.js:2
    at index.js? [sm]:1
    at require (VM282 WAService.js:2)
    at VM282 WAService.js:2
    at rules.js? [sm]:1
    at require (VM282 WAService.js:2)
errorReport @ VM282 WAService.js:2
thirdErrorReport @ VM282 WAService.js:2
(anonymous) @ VM282 WAService.js:2
I @ VM282 WAService.js:2
n @ VM282 WAService.js:2
(anonymous) @ VM282 WAService.js:2
value @ VM282 WAService.js:2
(anonymous) @ VM282 WAService.js:2
doWhenAllScriptLoaded @ VM407:1040
scriptLoaded @ VM407:1060
(anonymous) @ VM407:1085
error (async)
(anonymous) @ VM282 WAService.js:2
(anonymous) @ VM282 WAService.js:2
(anonymous) @ VM282 WAService.js:2
VM282 WAService.js:2 Error: Component is not found in path "miniprogram_npm/lin-ui/icon/index" (using by "miniprogram_npm/lin-ui/button/index")
    at A (VM282 WAService.js:2)

**Reply by juzi214032** [reply]
检查一下这个文件存在吗 miniprogram_npm/lin-ui/common/async-validator/validator/type.js

**Reply by jerryliushilin** [reply]
我新建了一个空白项目，只放入了裁剪组件，还是报错，微信开发工具是最新版，基础库是2.14.4
报错如下：
[渲染层网络层错误] Failed to load local image resource /miniprogram_npm/lin-ui/image-clipper/images/photo.png 
 the server responded with a status of 500 (HTTP/1.1 500 Internal Server Error) 

[渲染层网络层错误] Failed to load local image resource /miniprogram_npm/lin-ui/image-clipper/images/rotate-along.png 
 the server responded with a status of 500 (HTTP/1.1 500 Internal Server Error) 

[渲染层网络层错误] Failed to load local image resource /miniprogram_npm/lin-ui/image-clipper/images/rotate-inverse.png 
 the server responded with a status of 500 (HTTP/1.1 500 Internal Server Error) 

[渲染层网络层错误] Failed to load local image resource /miniprogram_npm/lin-ui/image-clipper/images/sure.png 
 the server responded with a status of 500 (HTTP/1.1 500 Internal Server Error) 

[渲染层网络层错误] Failed to load local image resource /miniprogram_npm/lin-ui/image-clipper/images/close.png 
 the server responded with a status of 500 (HTTP/1.1 500 Internal Server Error)

**Reply by juzi214032** [reply]
你电脑操作系统版本和开发者工具版本可以提供一下吗

**Comment by jerryliushilin** [comment]
我电脑操作系统是WIN10，64位专业版，开发工具版本号是1.03.2101150

发件人: 桔子 <notifications@github.com>
发送时间: 2021年1月21日 12:38
收件人: TaleLin/lin-ui <lin-ui@noreply.github.com>
抄送: 刘士麟 <liushilin@xdf.cn>; Author <author@noreply.github.com>
主题: Re: [TaleLin/lin-ui] 微信开发工具使用LIN-UI image-clipper组件时在PC端提示错误信息，在手机端可以正常使用 (#1258)


你电脑操作系统版本和开发者工具版本可以提供一下吗

**Comment by juzi214032** [comment, marked as answer]
**确认问题为微信开发者工具的问题，回退到 1.03.2011120 Stable 版本可正常使用**

1.03.2011120 Stable 下载地址：
- [Windows](https://dldir1.qq.com/WechatWebDev/release/p-ae42ee2cde4d42ee80ac60b35f183a99/wechat_devtools_1.03.2011120_x64.exe)
- [Mac OS](https://dldir1.qq.com/WechatWebDev/release/p-ae42ee2cde4d42ee80ac60b35f183a99/wechat_devtools_1.03.2011120.dmg)
